react/components/ListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react'
+import { Radio } from 'vtex.styleguide'
+import StorePickupItem from './StorePickupItem'
+
+import styles from './styles.css'
+
+interface Props {
+  isSelected: boolean
+  store: SkuPickupStore
+  onSelectItem: () => void
+}
+
+const ListItem: FC<Props> = ({ store, isSelected, onSelectItem }) => {
+  const bgColor = isSelected ? 'bg-near-white' : 'bg-base'
+  return (
+    <div className={`flex ${bgColor} pa4 bt bb b--muted-4 ph7-l ${styles.storeListItem}`}>
+      <div className="mt3">
+        <Radio
+          checked={isSelected}
+          id=""
```

Comment:
perhaps add a comment on what this radio is doing here? or why it's all blanked out? (something like `// Decorative radio element`)

Reply:
suggested above